desktop/lib/menu/editor-context-menu.js

```diff
@@ -0,0 +1,118 @@
+
+/**
+ * Original library:
+ * Author: Jeff Wear, Mixmax, Inc
+ * License: MIT
+ * From: https://github.com/mixmaxhq/electron-editor-context-menu
+ */
+
+var noop = function(){};
+var defaults = require('lodash.defaults');
+var isEmpty = require('lodash.isempty');
+var isFunction = require('lodash.isfunction');
+var isArray = require('lodash.isarray');
+var cloneDeep = require('lodash.clonedeep');
```

Comment:
I wonder if we should drop these lodash libs except for `clonedeep` since there are easy-ish alternatives.